### PR TITLE
fix: paragraph gets treated as comma

### PR DESCRIPTION
**Pull request by melzreal**
## Description
- In French keyboards, the browser is detecting the code for a paragraph to be a comma instead.
- To fix this, for the outlier of the comma,I'm using the check on key instead of code.

The remaining checks appear consistent across keyboards so I'm going with the minimum change possible

## Checklist

- [ ] :green_book: all commit messages follow the [conventional commits](https://conventionalcommits.org/) standard
- [ ] :arrow_left: changes are compatible with RTL direction
- [ ] :wheelchair: Changes to the UI are [tested for accessibility](./../README.md#accessibility-testing) and compliant with [WCAG 2.1](https://www.w3.org/TR/WCAG21/).
- [ ] :memo: changes are tested in Chrome, Firefox, Safari and Edge
- [ ] :iphone: changes are responsive and tested in mobile
- [ ] :+1: PR is approved by @zendesk/vikings

### CODE REVIEW

File: src/modules/new-request-form/fields/cc-field/useTagsInputContainer.ts [hunk 1]
```diff
@@ -90,12 +90,14 @@ export function useTagsInputContainer({
     const target = e.target as HTMLInputElement;
     const tag = target.value;
 
+    console.log(e.key);
```
Comment (luis-almeida) [inline src/modules/new-request-form/fields/cc-field/useTagsInputContainer.ts:93]: Nit: debug leftover?

**Review by luis-almeida — COMMENTED** [review]
Is it still draft?

File: src/modules/new-request-form/fields/cc-field/useTagsInputContainer.ts (hunk 1 above)
Comment (melzreal) [reply src/modules/new-request-form/fields/cc-field/useTagsInputContainer.ts]: was still in draft! I made the keys consistent now

**Comment by melzreal** [comment]
> Is it still draft?

it was when you checked x) I was just asking Gianluca if we were ok with the approach or wanted consistency, ended up going with consistency for the keys by referring to the Garden constants (cuz e.key for space its just an empty space so it looked a little weird)

**Review by luis-almeida — APPROVED** [review]

**Review by BrunoBFerreira — APPROVED** [review]

**Pull request merged**